### PR TITLE
[minor] provide default svg viewbox to fallback image

### PR DESCRIPTION
**Pull request by Swaagie**
Results in better alignment of the fallback image.

screen shot 2019-02-11 at 1 39 41 pm

Also works as expected in react-native, when multiple `Svg` components are used nested. 
screen shot 2019-02-12 at 12 58 40 pm

### CODE REVIEW

File: packages/provider/fallback/index.js [hunk 1]
```diff
@@ -18,10 +18,12 @@ export default function Fallback() {
 
   /*eslint-disable */
   return (
-    <G fill='#CCC'>
-      <Path d='M86,13.85H42a5,5,0,0,0-5,5v26.3a5,5,0,0,0,5,5H86a5,5,0,0,0,5-5V18.85A5,5,0,0,0,86,13.85Zm-44,2H86a3,3,0,0,1,3,3v23L73.33,26.18a1,1,0,0,0-1.41,0L58.2,39.92l-8-8a1,1,0,0,0-1.42,0L39,41.8V18.85A3,3,0,0,1,42,15.85Zm44,32.3H42a3,3,0,0,1-3-3v-.52L49.54,34.06,60.31,44.87a1,1,0,0,0,1.42,0,1,1,0,0,0,0-1.42l-2.11-2.12,13-13L89,44.63h0v.52A3,3,0,0,1,86,48.15Z'/>
-      <Path d='M58.21,30.07A5.24,5.24,0,1,0,53,24.84,5.24,5.24,0,0,0,58.21,30.07Zm0-8.47A3.24,3.24,0,1,1,55,24.84,3.24,3.24,0,0,1,58.21,21.6Z'/>
-    </G>
+    <Svg  width='100' height='100' viewBox='0 0 128 64'>
```
Comment (msluther) [inline packages/provider/fallback/index.js]: hyper-nit: there's an extra space between `Svg` and `width`. 😂 
```suggestion
    <Svg width='100' height='100' viewBox='0 0 128 64'>
```

File: packages/bundle/test/dimensions.test.js [hunk 2]
```diff
@@ -59,7 +59,7 @@ describe('dimensions', function () {
         assume(svg).equals(item);
 
         assume(item.viewBox).is.a('string');
-        assume(item.viewBox).equals('16.072994232177734 55.62681579589844 493.87457275390625 509.17413330078125');
+        assume(item.viewBox).equals('0 0 0 0');
```
Comment (3rd-Eden) [inline packages/bundle/test/dimensions.test.js]: This seems really weird, and no way this is valid. It's getting height and width of 0.

Comment (Swaagie) [reply packages/bundle/test/dimensions.test.js]: how is this related to what I'm doing though? I'll investigate..

Comment (Swaagie) [reply packages/bundle/test/dimensions.test.js]: fixed with latest commit, puppeteer hack is no longer need due to upstream fixes https://github.com/GoogleChrome/puppeteer/issues/728#issuecomment-352005352

**Pull request merged**